spec/nanaimo/reader_spec.rb

```diff
@@ -107,6 +107,19 @@ module Nanaimo
           expect(subject).to eq Nanaimo::Dictionary.new({ Nanaimo::String.new('key', '') => Nanaimo::String.new('$PROJECT_DIR/mogenerator/mogenerator', '') }, '')
         end
       end
+
+      describe 'that contain' do
+        valid_characters = ('a'..'z').to_a + ('A'..'Z').to_a + ('0'..'9').to_a + %w(_ $ / : . -)
```

Comment:
👍 - Looks good. I'd also probably delete the test at https://github.com/CocoaPods/Nanaimo/pull/14/files#diff-e4dbcff8a21097d33c0121cb86a78d8aL103 because it's redundant now.